pl-l2d44c1/pl-l2d44c1.xml

```diff
@@ -54,41 +54,93 @@
-          nitatem- perdidit voluntatis, ex se ipsa rursus eam habere, potuis- set, inulto
-          possibilius hoc natura haberet angelica, quae quanto mi- nus gravatur terreni corporis
-          pondere, tanto 'magis hac esset prae- dita facultate 1 ». Non igitur homo vel Angelus a se
+          Deo nobis est, sed a nobis et a diabolo, bona autem a Deo tantum nobis est.
+          <cit>
+            <quote>
+            Bonae namque voluntatis et cogitationis initium non homini ex se ipso nasci, 
+            sed divinitus parari et tribui, in eo Deus evidenter ostendit, quia nec diabolus nec aliquis
+            Angelorum eius, ex quo in hanc caliginem sunt detrusi, bonam potuit vel poterit resumere
+            voluntatem; quia, si possibile foret, ut humana natura, postquam a Deo aversa, bonitatem
+            perdidit voluntatis, ex se ipsa rursus eam habere, potuisset, multo
+            possibilius hoc natura haberet angelica, quae quanto minus gravatur terreni corporis
+            pondere, tanto magis hac esset praedita facultate
+            <quote>
+          <cit>
+          . Non igitur homo vel Angelus a se
           voluntatem bonam habere potest, sed malam. Similiter et de potentia inquiunt, per
-          similitudinem voluntatis de potentia boni et mali disserentes b, quod illa sit a Deo, non
+          similitudinem voluntatis de potentia boni et mali disserentes, quod illa sit a Deo, non
           ista. </p>
+          <!--this is a closed quote. Maybe it doesn't like the cit and quote inside?-->
+
         <p xml:id="pl-l2d44c1-d1e116"> Sed pluribus Sanctorum testimoniis indubitanter monstratur, quod potestas mali a Deo
           est, a quo est omnis potestas. </p>
-        <p xml:id="pl-l2d44c1-d1e120"> Ait enim Apostolus <quote xml:id="pl-l2d44c1-Qd1e122">Non est potestas nisi a Deo</quote>; quod non de: potestate
-          boni tantum, sed et, mali intelligi oportet, cum Pilato d- Veritas dicat3: Non haberes in
-          me potestatem, nisi tta- ' tam esset tibi 6 desuper. </p>
-        <p xml:id="pl-l2d44c1-d1e126"> « Malitia nempe hominum, ut ait Augusti n us 4, cupiditatem nocendi per se habet;
-          potestatem autem, si ille non dat, non ha; bet». ideoque diabolus, antequam
-          aliquidtolleret lob, dicebat Dominoö: Mitte manum tuam. 7", id est, da potestatem, «quia
-          etiam nocentium potestas non est nisi a Deo, sicut Sapientia aitG: Per me reges regnant,
-          et tyranni per me tenent terrain. Unde et 9 lob 7 de Domino ait: Qai facit regnare
-          hypocritam propter perversitatem h, populi. Et de populo. Israel dicit DeusS: Dedi eis
-          regem in ira mea 9 ». « Nocendi enim voluntas potest esse ab animo hominis, potestas autem
+
+        <p xml:id="pl-l2d44c1-d1e120"> Ait enim 
+        <ref>
+          <name>Apostolus<name>
+          <!--this is a reference to Paul since it's quoting Romans, but not sure if it deserves a name tag-->
```

Comment:
yes `<name>` tag inside `<ref>` is correct here, but `Apostolus` is a special way they refer to the apostle paul. When the say "the apostle", they always mean Paul